src/main/resources/static/js/admin.js

```diff
@@ -44,34 +44,40 @@ form.addEventListener('submit', (event) => {
     createProduct(product);
 });
 
-// TODO: [1단계] 상품 관리 CRUD API에 맞게 변경
 const createProduct = (product) => {
+    const {name, imgUrl, price} = product;
+
     axios.request({
-        url: '',
+        url: '/products',
+        method: 'POST',
+        data: {
+            "name": name, "imgUrl": imgUrl, "price": price
+        }
     }).then((response) => {
         window.location.reload();
     }).catch((error) => {
         console.error(error);
     });
 };
 
-// TODO: [1단계] 상품 관리 CRUD API에 맞게 변경
 const updateProduct = (product) => {
-    const { id } = product;
+    const {id, name, imgUrl, price} = product;
 
     axios.request({
-        url: '',
+        url: '/products/' + id, method: 'PATCH', data: {
```

Comment:
```suggestion
url: '/products/' + id,
method: 'PATCH',
data: {
  "name": name,
  "imgUrl": imgUrl,
  "price": price
}
```

POST와 일관성있게 가면 좋을 것 같아요 :)

Reply:
@woo-chang 

자동으로 자꾸 줄을 합치네...! 
일관성 가릿